Dockerfile

```diff
@@ -22,6 +22,17 @@ RUN go build -a -installsuffix cgo \
     -ldflags="-X github.com/cyberark/conjur-authn-k8s-client/pkg/authenticator.Tag=$TAG" \
     -o authenticator ./cmd/authenticator
 
+# verify the binary is using BoringCrypto.
+# Outputting to /dev/null so the output doesn't include all the files
+RUN sh -c "go tool nm authenticator | grep '_Cfunc__goboringcrypto_' 1> /dev/null"
```

Comment:
I don't know for sure but it would be good to check if `sh` fails when grep fails too (in `bash` you need both `set -e` and `set -o pipefail` for this line to work as expected).

Reply:
yes, i verified it. the output will be:
```
Step 12/29 : RUN sh -c "go tool nm authenticator | grep '_Cfunc__goboringcrypto_' 1> /dev/null"
 ---> Running in 9711b93a98f9
The command '/bin/sh -c sh -c "go tool nm authenticator | grep '_Cfunc__goboringcrypto_' 1> /dev/null"' returned a non-zero code: 1
```

Reply:
and the build will fail of course.